File: BHAVRP_Python/cujae/inf/citi/om/problem/input/customer.py
from .location import Location

class Customer:
    # Constructor para la clase Customer.
    def __init__(
        self, 
        id_customer: int, 
        request_customer: float, 
        location_customer: Location
    ):
        self.id_customer = id_customer
        self.request_customer = request_customer
        self.location_customer = location_customer
    
    def __init__(self):
        self.id_customer = None
        self.request_customer = 0.0
        self.location_customer = None
    
    def get_id_customer(self) -> int:
        return self.id_customer

    def set_id_customer(self, value: int):
        self.id_customer = value

    def get_request_customer(self) -> float:
        return self.request_customer

    def set_request_customer(self, value: float):
        self.request_customer = value

    def get_location_customer(self) -> Location:
        return self.location_customer

    def set_location_customer(self, value: Location):
        self.location_customer = value

File: BHAVRP_Java/BHAVRP2.0/src/cujae/inf/ic/om/problem/input/Customer.java
package cujae.inf.ic.om.problem.input;

public class Customer {
	private int idCustomer;
	private double requestCustomer;
	private Location locationCustomer;
	
	public Customer() {
		super();
	}
	
	public Customer(int idCustomer, double requestCustomer,
			Location locationCustomer) {
		super();
		this.idCustomer = idCustomer;
		this.requestCustomer = requestCustomer;
		this.locationCustomer = locationCustomer;
	}

	public int getIDCustomer() {
		return idCustomer;
	}

	public void setIDCustomer(int idCustomer) {
		this.idCustomer = idCustomer;
	}

	public double getRequestCustomer() {
		return requestCustomer;
	}

	public void setRequestCustomer(double requestCustomer) {
		this.requestCustomer = requestCustomer;
	}

	public Location getLocationCustomer() {
		return locationCustomer;
	}

	public void setLocationCustomer(Location locationCustomer) {
		this.locationCustomer = locationCustomer;
	}
}